bpf/lib/common.h

```diff
@@ -84,11 +84,13 @@ enum {
 #define CILIUM_CALL_IPV4_FROM_LXC		7
 #define CILIUM_CALL_IPV4_FROM_NETDEV		CILIUM_CALL_IPV4_FROM_LXC
 #define CILIUM_CALL_IPV4_FROM_OVERLAY		CILIUM_CALL_IPV4_FROM_LXC
+#define CILIUM_CALL_IPV4_FROM_WIREGUARD		CILIUM_CALL_IPV4_FROM_LXC
```

Comment:
Even though we'd perform function call rather than tail call in `tail_handle_ipv{4,6}`, these are still needed for the nodeport codepath. Otherwise, a "potential missed tail call" will be hit.